avro-fastserde/src/main/java/com/linkedin/avro/primitive/PrimitiveGenericRecord.java

```diff
@@ -0,0 +1,103 @@
+package com.linkedin.avro.primitive;
```

Comment:
nit - if we're gonna go extending avro ... "com.linkedin.avro.extensions" ?

Reply:
Sure, or maybe `com.linkedin.avro.api`?

Reply:
works for me